landmark_gnss:
    - [367933.061032, 3955734.62339]
    - [367957.18309, 3955741.34757]
goal:
    - [1.0, 2.0]
    - [3.0, 4.0]
    - [1.0, 2.0]
    - [3.0, 4.0]
